vnext/Microsoft.ReactNative.Cxx/CppWinRTIncludes.h

```diff
@@ -10,6 +9,8 @@
 
 #ifndef USE_WINUI3
 
+#pragma include_alias("winrt/impl/Windows.UI.Xaml.2.h", "UI.Xaml.impl.override.h")
```

Comment:
why do we need this?

Reply:
I t was an attempt to learn how we can augment or override the ABI consumer code generated by C++/WinRT. I will remove it here.

---
In reply to: [432992926](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r432992926) [](ancestors = 432992926)

Reply:
Removed.

---
In reply to: [433256201](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433256201) [](ancestors = 433256201,432992926)